pkg/user/api/validation/validation.go

```diff
@@ -10,16 +11,35 @@ import (
 
 func ValidateUserName(name string, prefix bool) (bool, string) {
 	if strings.Contains(name, "%") {
-		return false, `Usernames may not contain "%"`
+		return false, `may not contain "%"`
 	}
 	if strings.Contains(name, "/") {
-		return false, `Usernames may not contain "/"`
+		return false, `may not contain "/"`
+	}
+	if strings.Contains(name, ":") {
+		return false, `may not contain ":"`
 	}
 	if name == ".." {
-		return false, `Usernames may not equal ".."`
+		return false, `may not equal ".."`
 	}
 	if name == "." {
-		return false, `Usernames may not equal "."`
+		return false, `may not equal "."`
+	}
+	return true, ""
+}
+
+func ValidateIdentityName(name string, prefix bool) (bool, string) {
```

Comment:
signature required for use with kvalidation methods, changed to `_` to indicate deadness